python/pyarrow/fs.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import absolute_import
+
+from pyarrow._fs import *  # noqa
```

Comment:
I think it's nicer to list the imports explicitly. There aren't many of them.

Reply:
IMO we should move it to `fs.pyx` instead of proxying through another python file.

Reply:
I think it's beneficial to have the `_$NAME.pyx` and `$NAME.py` pattern, so that we aren't compiling code we don't need to